### PR TITLE
Drop export of implementations & export Implementation class.

### PR DESCRIPTION
**Pull request by aljones15**

### CODE REVIEW

File: lib/main.js [hunk 1]
```diff
@@ -4,10 +4,11 @@
 import {Implementation} from './Implementation.js';
 import {implementerFiles} from '../implementations/index.js';
 
+export {Implementation};
+
 const keyValues = implementerFiles.map(
   implementation => [implementation.name, new Implementation(implementation)]);
-export const implementations = new Map(keyValues);
-const allImplementations = implementations;
+export const allImplementations = new Map(keyValues);
```
Comment (JSAssassin) [inline lib/main.js:11]: Can we name this `implementations` instead of `allImplementations` so it doesn't become a breaking change?

Comment (aljones15) [reply lib/main.js]: I prefer `implementations` too, so I've decided to just export both `implementations` and `allImplementations` until we do a major release and consensus on that naming convention occurs.

File: test/Example.spec.js [hunk 2]
```diff
@@ -4,10 +4,10 @@
 import chai from 'chai';
 const should = chai.should();
 
-import {implementations} from '../lib/main.js';
+import {allImplementations} from '../lib/main.js';
 
 describe('implementations', () => {
   it('should exist', async () => {
```
Comment (JSAssassin) [inline test/Example.spec.js:10]: Should additionally add back the test that checks if  `implementations` exist.

**Review by JSAssassin — APPROVED** [review]
LGTM!

File: test/Example.spec.js (hunk 2 above)
Comment (aljones15) [reply test/Example.spec.js:10]: I think we're ok as is here. We do need to expand the tests.

**Pull request merged**